assets/style.styl

```diff
@@ -228,18 +243,21 @@ for $name, $motion in $animBank
     animation-duration $animDuration
     animation-timing-function $animEasingOut
   @keyframes fadeIn-{$name}
-    from
-      transform $motion scale(.8)
+    0%, 20%
```

Comment:
For a small amount of time, the animation will exist hidden in it's final state so we don't have to account for the transforms in the React component and get final values immediately.